UVicMartletplace/martletplace
# Gavin/playing with lat and lon stuff

# Description
Changing latitude and longitude as follows:

(previously)
"location": { "latitude": 45.4215, "longitude": -75.6972 },

(after change)
"location": { "lat": 40.7128, "lon": -74.006 }.

This makes the frontend search by location within 5km of input work correctly, and sorting by distance ascending by descending.

![image](https://github.com/user-attachments/assets/62dfc3b5-09cb-4f18-bbd0-52aa0e18c42c)
![image](https://github.com/user-attachments/assets/11fd35d6-9943-4179-a585-aac595740bbb)

I tested the app manually and ran backend tests and couldn't find anything broken by this.

Closes #260 

## How to Test
- Build the app, log in, search listings, and sort by location (it should change the order of listings)
- Also try creating a listing at your current location, then search for it and sort by ascending location (it should appear first)
- Once the application is built, run `docker exec -it martletplace_search bash` in a different terminal, then run `pytest src/test_server.py`

## Checklist
